src/viur_cli/cloud.py

```diff
@@ -253,7 +253,7 @@ def transform_dict_to_yaml(transformed_data):
 
 
 def gcloud_setup():
-    project = input("Enter PROJECT_ID").strip()
+    project = input("Enter PROJECT_ID: ").strip()
 
     if not project:
         echo_fatal("Usage: python scrtipt.py PROJECT_ID")
```

Comment:
btw: What's _scrtipt.py_?

Shouldn't this be sth like
```suggestion
        echo_fatal("Usage: viur setup gcloud PROJECT_ID")
```
?